Assets/Scripts/Enemies/Bosses/SpikeBall.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
+
+public class SpikeBall : EnemyMovement
+{
+
+    float timer = 0;
+
+    public override Vector3 PositionTransform(EnemyPosition position, float deltaTime)
+    {
+        return position.Up * deltaTime;
+    }
+
+    public override Vector3 RotationTransform(EnemyPosition position, float deltaTime)
+    {
+        return Vector3.zero;
+    }
+
+    public override void Update()
+    {
+        base.Update();
```

Comment:
I think a cleaner solution would to be add another abstract method to `EnemyMovement` `public abstract void OnUpdate(EnemyPosition position, float deltaTime)` and then call that on update in `EnemyContoller` https://github.com/fullerga/Dodge/blob/SpikeBall/Assets/Scripts/Enemies/EnemyController.cs#L32